### PR TITLE
docs(inputs.knx_listener): add a link to related project to ease config generation for KNX Telegraf Input Plugin

### PR DESCRIPTION
**Pull request by svsool**
# Required for all PRs

- [x] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [ ] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

👋 Adds a link to related tool to improve discoverability a bit and ease config generation for people who use KNX Telegraf Input Plugin.

### CODE REVIEW

**Comment by svsool** [comment]
!signed-cla

File: plugins/inputs/knx_listener/README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   #   addresses = ["5/5/3"]
 ```
 
+### Related tools
+
+- [knx-influx-config-generator](https://github.com/svsool/knx-influx-config-generator) generates configuration from KNX project file
```
Comment (powersj) [inline plugins/inputs/knx_listener/README.md:57]: Hi,

Cool tool! I think we would consider adding this to the docs, but I have two questions/asks:

1) This project is called telegraf, not influx. Generating an "influx" config is a bit confusing as that isn't the project name or what we refer to the config file as.
2) Can you please put an open source licence in your project. It is your tool so you can pick any of them, but I would prefer we link to something that has one specified.

Thanks!

Comment (svsool) [reply plugins/inputs/knx_listener/README.md]: hey @powersj, let me know if it's better now, thanks!

Comment (powersj) [reply plugins/inputs/knx_listener/README.md]: perfect! thank you so much!

**Review by powersj — APPROVED** [review]

**Review by srebhan — APPROVED** [review]
Very cool project @svsool!

**Pull request merged**